pointing_component/sal3.9_dev/ATPtg.info

```diff
@@ -34,7 +34,7 @@ instrument_location "Nasmyth left"
     instrument_alignment_angle 0.0
 }
 "Nasmyth right" {
-    instrument_alignment_angle 0.0
+    instrument_alignment_angle 180.0
```

Comment:
General comment on the pointing files. Are we developing in SAL 3.9 here or do we need to think about moving things to a SAL agnostic location?

Reply:
yeah no... long time ago I had a hard time making this builds SAL agnostics. I think we got it to a point where this is stable enough though. I can rename the directory if you'd like

Reply:
I would like it renamed, but we don't need to do it right now unless you want to. Otherwise, file a ticket for it.